search-service/src/main/kotlin/com/egm/stellio/search/service/EntityPayloadService.kt

```diff
@@ -103,6 +103,36 @@ class EntityPayloadService(
             .bind("specific_access_policy", specificAccessPolicy?.toString())
             .execute()
 
+    @Transactional
+    suspend fun mergeEntity(
+        entityId: URI,
+        ngsiLdAttributes: List<NgsiLdAttribute>,
+        expandedAttributes: ExpandedAttributes,
+        observedAt: ZonedDateTime?,
+        sub: Sub?
+    ): Either<APIException, UpdateResult> = either {
+        val createdAt = ZonedDateTime.now(ZoneOffset.UTC)
```

Comment:
look more like a `modifiedAt` than a `createdAt`

Reply:
also you could use the `ngsildDateTime()` function?

Reply:
It's a modifiedAt when we modify attributes which exist but when we create new attributes it's createdAt

Reply:
for the replace entity endpoint you implemented just before, it is called `replacedAt`. so I would suggest to call it `mergedAt` here (it avoids confusions)

Reply:
ok i will change that